content/case_histories/balboa/data.rst

```diff
@@ -0,0 +1,13 @@
+.. _balboa_data:
+
+Data
+====
+
+:numref:`XXX4` presents the ZTEM survey flight lines over the DEM (digital elevation model) and the aeromagnetic signature of the Cobre Panama region, as well as known porphyry deposit outlines and drill hole locations prior to the discovery of Balboa in 2010. The figure shows a correlation between relatively higher elevations and magnetic high anomalies over deposit area. The magnetic RTP (reduced-to-pole) image in Figure 4b shows a recognizable pattern of magnetic highs from volcanic andesites due to primary magnetite (Burge, 2014). However, all the known porphyry deposits are noticeably centred on magnetic lows, possibly representing demagnetized areas due to phyllic alteration (Burge, 2014). Exceptionally, Balboa coincides with a positive magnetic anomaly, which, along with the lack of geochemical anomaly due to thick leach-cap (G. Wells, pers. comm., 01-2016), masked it in previous exploration phases (Fiscor, 2014).
```

Comment:
- [ ] could we use more descriptive references and names for figures?
- [ ] also, we will need to add the references to the bibtex file (then we can do a find-replace for the text so that they are linked properly)